SignalServiceKit/src/Messages/OWSReadReceiptManager.m

```diff
@@ -445,6 +412,148 @@ - (void)updateOutgoingMessageFromLinkedDevice:(TSOutgoingMessage *)message
     [TSRecipientReadReceipt removeRecipientIdsForTimestamp:message.timestamp transaction:transaction];
 }
 
+#pragma mark - Linked Device Read Receipts
+
+- (void)updateIncomingMessage:(TSIncomingMessage *)message
```

Comment:
nit: this is kind of vague - maybe `applyAnyEarlyReadReceiptsForIncomingMessage:`

TIOLI

Reply:
Did something similar.